src/main/resources/ja/ancient.conf

```
@@ -0,0 +1,3 @@
+ancient {
+  gods: ["アフロディテ", "アポロン", "アレス", "アルテミス", "アテナ", "デメテル", "ディオニュソス", "ハデス", "ヘパイストス", "ヘラ", "ヘルメス", "ヘスティア", "ポセイドン", "ゼウス"]
```

Let's be consistent with syntax and use `=` here
```suggestion
  gods = ["アフロディテ", "アポロン", "アレス", "アルテミス", "アテナ", "デメテル", "ディオニュソス", "ハデス", "ヘパイストス", "ヘラ", "ヘルメス", "ヘスティア", "ポセイドン", "ゼウス"]
```